### ISSUE TITLE
Maybe a good way for prediction.

### ISSUE THREAD

**Issue by easonnie** [dennybritz/cnn-text-classification-tf#33]
In the code segment below.
`self.h_pool_flat = tf.reshape(self.h_pool, [-1, num_filters_total])`
`self.h_drop = tf.nn.dropout(self.h_pool_flat, self.dropout_keep_prob)`
`self.scores = tf.nn.xw_plus_b(self.h_drop, W, b, name="scores")
            self.predictions = tf.argmax(self.scores, 1, name="predictions")`
The predictions operation do not automatically leave the dropout.

Maybe a good way for prediction:
`self.predictions = tf.argmax(tf.nn.xw_plus_b(self.h_pool_flat, W, b), 1, name="predictions")`
In this way, we do not need to feed the keep_rate=1.0 every time we make a prediction and the loss function will not be affected.
